docs/content/latest/develop/graphql/hasura.md

```diff
@@ -11,21 +11,23 @@ isTocNested: true
 showAsideToc: true
 ---
 
-Integrate the [Hasura GraphQL engine](https://hasura.io) with YugabyteDB to use GraphQL on your YugabyteDB databases and applications.
+Use the [Hasura GraphQL Engine](https://hasura.io) with YugabyteDB to use GraphQL with your YugabyteDB databases and applications.
 
-Follow the steps below to learn how easily you can begin using the Hasura GraphQL engine with YugabyteDB. For details on using the Hasura GraphQL engine, see the [Hasura GraphQL engine documentation](https://docs.hasura.io).
+Follow the steps below to learn how easily you can begin using the Hasura GraphQL Engine with YugabyteDB. For details on using the Hasura GraphQL engine, see the [Hasura GraphQL engine documentation](https://docs.hasura.io).
 
 ## Before you begin
 
 ### Install and start YugabyteDB
 
-Before starting and running YugabyteDB with Hasura, you need to add the YugabyteDB environment variable `YB_SUPPRESS_UNSUPPORTED_ERROR=1`. Setting the value to 1 suppresses unsupported error exceptions and raise only warnings. To set the environment variable , run the following command.
+To use the Hasura GraphQL Engine with YugabyteDB, you need to set the `yb-tserver` option `--ysql_suppress_unsupported_error` to `true` so that errors on the use of unsupported SQL statements and return only warnings instead.
+
+If you're using `yb-ctl` to start your cluster, you can add the option like this:
 
 ```sh
-$ export YB_SUPPRESS_UNSUPPORTED_ERROR=1
+$ ./bin/yb-ctl start --tserver_flags "ysql_suppress_unsupported_error=true"
 ```
 
-If you're new to YugabyteDB, you can be up and running with YugabyteDB in under five minutes by following the steps in [Quick start](https://docs.yugabyte.com/latest/quick-start/).
+If you are new to YugabyteDB, you can be up and running with YugabyteDB in under five minutes by following the steps in [Quick start](https://docs.yugabyte.com/latest/quick-start/).
```

Comment:
This is tricky - ideally, quick start should come before the statement above which talks about passing the gflag in yb-ctl.

Reply:
I thought about that -- I'll move it.